versioned/storage/store/src/main/java/org/projectnessie/versioned/storage/versionstore/TransplantIndividualImpl.java

```diff
@@ -114,8 +114,9 @@ public MergeResult<Commit> transplant(Optional<?> retryState, TransplantOp trans
       empty = false;
       if (!transplantOp.dryRun()) {
         newHead = newCommit.id();
-        boolean committed = commitLogic.storeCommit(newCommit, objsToStore);
-        if (committed) {
+        boolean fastForward = sourceCommit.id().equals(newHead);
+        if (!fastForward) {
```

Comment:
In case of fast forward, the new commit is identical to the source commit and doesn't need to be persisted again. 
Unfortunately we cannot call `CommitObj.equals` because the two commits actually differ by one property: `CommitObj.created()`.
This begs the question: should `CommitObj.created()` be annotated with `@Value.Auxiliary`?